packages/legacy-client/src/client/objectSets/OsdkObjectSet.ts

```diff
@@ -34,19 +35,22 @@ import { createObjectSetAggregationStep } from "./createObjectSetAggregationStep
 import { createObjectSetOrderByStep } from "./createObjectSetOrderByStep";
 import { createObjectSetSearchAround } from "./createObjectSetSearchAround";
 import { createObjectSetTerminalLoadStep } from "./createObjectSetTerminalLoadStep";
+import { mapPropertiesToSearchFilter } from "./mapPropertiesToSearchFilter";
 
 export function createOsdkObjectSet<
   O extends OntologyDefinition<any>,
   K extends ObjectTypesFrom<O>,
 >(
   clientContext: ClientContext,
+  apiName: K,
   objectSetDefinition: ObjectSetDefinition,
   ontologyDefinition: O,
```

Comment:
OntologyDefinition already lives in ClientContext, should we make it ClientContext<O> so that you can just use the fully type inferred ontology definition? (Future PR, I can do it)

Reply:
Yeah I actually realized we have it passed in three times when I was writing the test, the thin client, here, and context 🙈